<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Eggs3</class>
 <widget class="QWidget" name="Eggs3">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1200</width>
    <height>700</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Gen 3 Eggs</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QGroupBox" name="groupBoxProfile">
     <property name="title">
      <string>Profile</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_13">
      <item row="1" column="3">
       <widget class="QLabel" name="labelProfileSIDValue">
        <property name="text">
         <string notr="true">54321</string>
        </property>
       </widget>
      </item>
      <item row="0" column="3">
       <widget class="QLabel" name="labelProfileTIDValue">
        <property name="text">
         <string notr="true">12345</string>
        </property>
       </widget>
      </item>
      <item row="0" column="4" rowspan="2">
       <widget class="Line" name="line">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
       </widget>
      </item>
      <item row="0" column="6">
       <widget class="QLabel" name="labelProfileGameValue">
        <property name="text">
         <string>Emerald</string>
        </property>
       </widget>
      </item>
      <item row="0" column="5">
       <widget class="QLabel" name="labelProfileGame">
        <property name="text">
         <string>Game</string>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QLabel" name="labelProfileSID">
        <property name="text">
         <string>SID</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QPushButton" name="pushButtonProfileManager">
        <property name="text">
         <string>Manager</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="labelProfile">
        <property name="text">
         <string>Profile</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QComboBox" name="comboBoxProfiles"/>
      </item>
      <item row="0" column="2">
       <widget class="QLabel" name="labelProfileTID">
        <property name="text">
         <string>TID</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QTabWidget" name="tabEggSelection">
     <widget class="QWidget" name="tabEmerald">
      <attribute name="title">
       <string>Emerald</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_17">
       <item row="1" column="0" colspan="3">
        <widget class="TableView" name="tableViewEmerald"/>
       </item>
       <item row="0" column="0">
        <widget class="QGroupBox" name="groupBoxEmeraldRNGInfo">
         <property name="title">
          <string>RNG Info</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_8">
          <item row="0" column="0" colspan="2">
           <layout class="QGridLayout" name="gridLayout_2">
            <item row="0" column="0">
             <widget class="QLabel" name="labelEmeraldAdvanceHeld">
              <property name="text">
               <string>Held Advances</string>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="TextBox" name="textBoxEmeraldInitialAdvancesHeld">
              <property name="text">
               <string notr="true">1000</string>
              </property>
             </widget>
            </item>
            <item row="0" column="2">
             <widget class="TextBox" name="textBoxEmeraldMaxAdvancesHeld">
              <property name="text">
               <string notr="true">5000</string>
              </property>
             </widget>
            </item>
            <item row="1" column="0">
             <widget class="QLabel" name="labelEmeraldPickupAdvance">
              <property name="text">
               <string>Pickup Advances</string>
              </property>
             </widget>
            </item>
            <item row="1" column="1">
             <widget class="TextBox" name="textBoxEmeraldInitialAdvancesPickup">
              <property name="text">
               <string notr="true">1000</string>
              </property>
             </widget>
            </item>
            <item row="1" column="2">
             <widget class="TextBox" name="textBoxEmeraldMaxAdvancesPickup">
              <property name="text">
               <string notr="true">5000</string>
              </property>
             </widget>
            </item>
            <item row="2" column="0">
             <widget class="QLabel" name="labelEmeraldDelay">
              <property name="text">
               <string>Delay (Held / Pickup)</string>
              </property>
             </widget>
            </item>
            <item row="2" column="1">
             <widget class="TextBox" name="textBoxEmeraldDelayHeld"/>
            </item>
            <item row="2" column="2">
             <widget class="TextBox" name="textBoxEmeraldDelayPickup"/>
            </item>
            <item row="3" column="0">
             <widget class="QLabel" name="labelEmeraldCalibration">
              <property name="text">
               <string>Calibration</string>
              </property>
             </widget>
            </item>
            <item row="3" column="1" colspan="2">
             <widget class="TextBox" name="textBoxEmeraldCalibration">
              <property name="text">
               <string notr="true">18</string>
              </property>
             </widget>
            </item>
            <item row="4" column="0">
             <widget class="QLabel" name="labelEmeraldRedraws">
              <property name="text">
               <string>Redraws</string>
              </property>
             </widget>
            </item>
            <item row="4" column="1">
             <widget class="TextBox" name="textBoxEmeraldMinRedraws">
              <property name="text">
               <string notr="true">0</string>
              </property>
             </widget>
            </item>
            <item row="4" column="2">
             <widget class="TextBox" name="textBoxEmeraldMaxRedraws">
              <property name="text">
               <string notr="true">5</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="labelEmeraldMethod">
            <property name="text">
             <string>Method</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="ComboBox" name="comboBoxEmeraldMethod">
            <item>
             <property name="text">
              <string>Normal</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Split</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Alternate</string>
             </property>
            </item>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="labelEmeraldCompatibility">
            <property name="text">
             <string>Compatibility</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="ComboBox" name="comboBoxEmeraldCompatibility">
            <item>
             <property name="text">
              <string>The two don't seem to like each other</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>The two seem to get along</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>The two seem to get along very well</string>
             </property>
            </item>
           </widget>
          </item>
          <item row="3" column="0" colspan="2">
           <widget class="QPushButton" name="pushButtonEmeraldGenerate">
            <property name="text">
             <string>Generate</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="0" column="2">
        <widget class="QGroupBox" name="groupBoxEmeraldFilters">
         <property name="title">
          <string>Filters</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_4">
          <item row="0" column="0">
           <widget class="Filter" name="filterEmerald" native="true"/>
          </item>
         </layout>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QGroupBox" name="groupBoxEmeraldSettings">
         <property name="title">
          <string>Settings</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_5">
          <item row="0" column="0">
           <widget class="EggSettings" name="eggSettingsEmerald" native="true"/>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tabRSFRLG">
      <attribute name="title">
       <string>RS/FRLG</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_15">
       <item row="0" column="0">
        <widget class="QGroupBox" name="groupBoxRSFRLGRNGInfo">
         <property name="title">
          <string>RNG Info</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_11">
          <item row="0" column="0" colspan="2">
           <layout class="QGridLayout" name="gridLayout_6">
            <item row="0" column="2">
             <widget class="TextBox" name="textBoxRSFRLGSeedPickup">
              <property name="text">
               <string notr="true"/>
              </property>
             </widget>
            </item>
            <item row="1" column="2">
             <widget class="TextBox" name="textBoxRSFRLGMaxAdvancesHeld">
              <property name="text">
               <string notr="true">5000</string>
              </property>
             </widget>
            </item>
            <item row="1" column="1">
             <widget class="TextBox" name="textBoxRSFRLGInitialAdvancesHeld">
              <property name="text">
               <string notr="true">1000</string>
              </property>
             </widget>
            </item>
            <item row="2" column="1">
             <widget class="TextBox" name="textBoxRSFRLGInitialAdvancesPickup">
              <property name="text">
               <string notr="true">1000</string>
              </property>
             </widget>
            </item>
            <item row="2" column="0">
             <widget class="QLabel" name="labelRSFRLGPickupAdvance">
              <property name="text">
               <string>Pickup Advances</string>
              </property>
             </widget>
            </item>
            <item row="2" column="2">
             <widget class="TextBox" name="textBoxRSFRLGMaxAdvancesPickup">
              <property name="text">
               <string notr="true">5000</string>
              </property>
             </widget>
            </item>
            <item row="0" column="0">
             <widget class="QLabel" name="labelRSFRLGSeed">
              <property name="text">
               <string>Seed (Held / Pickup)</string>
              </property>
             </widget>
            </item>
            <item row="1" column="0">
             <widget class="QLabel" name="labelRSFRLGAdvanceHeld">
              <property name="text">
               <string>Held Advances</string>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="TextBox" name="textBoxRSFRLGSeedHeld">
              <property name="text">
               <string notr="true"/>
              </property>
             </widget>
            </item>
            <item row="3" column="0">
             <widget class="QLabel" name="labelRSFRLGDelay">
              <property name="text">
               <string>Delay (Held / Pickup)</string>
              </property>
             </widget>
            </item>
            <item row="3" column="1">
             <widget class="TextBox" name="textBoxRSFRLGDelayHeld"/>
            </item>
            <item row="3" column="2">
             <widget class="TextBox" name="textBoxRSFRLGDelayPickup"/>
            </item>
           </layout>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="labelRSFRLGMethod">
            <property name="text">
             <string>Method</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="ComboBox" name="comboBoxRSFRLGMethod">
            <item>
             <property name="text">
              <string>Normal</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Split</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Alternate</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Mixed</string>
             </property>
            </item>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="labelRSFRLGCompatibility">
            <property name="text">
             <string>Compatibility</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="ComboBox" name="comboBoxRSFRLGCompatibility">
            <item>
             <property name="text">
              <string>The two don't seem to like each other</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>The two seem to get along</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>The two seem to get along very well</string>
             </property>
            </item>
           </widget>
          </item>
          <item row="3" column="0" colspan="2">
           <widget class="QPushButton" name="pushButtonRSFRLGGenerate">
            <property name="text">
             <string>Generate</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QGroupBox" name="groupBoxRSFRLGSettings">
         <property name="title">
          <string>Settings</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_7">
          <item row="0" column="0">
           <widget class="EggSettings" name="eggSettingsRSFRLG" native="true"/>
          </item>
         </layout>
        </widget>
       </item>
       <item row="0" column="2">
        <widget class="QGroupBox" name="groupBoxRSFRLGFilters">
         <property name="title">
          <string>Filters</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_3">
          <item row="0" column="0" rowspan="3" colspan="2">
           <widget class="Filter" name="filterRSFRLG" native="true"/>
          </item>
         </layout>
        </widget>
       </item>
       <item row="1" column="0" colspan="3">
        <widget class="TableView" name="tableViewRSFRLG"/>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>TextBox</class>
   <extends>QLineEdit</extends>
   <header>Form/Controls/TextBox.hpp</header>
  </customwidget>
  <customwidget>
   <class>TableView</class>
   <extends>QTableView</extends>
   <header>Form/Controls/TableView.hpp</header>
  </customwidget>
  <customwidget>
   <class>ComboBox</class>
   <extends>QComboBox</extends>
   <header>Form/Controls/ComboBox.hpp</header>
  </customwidget>
  <customwidget>
   <class>Filter</class>
   <extends>QWidget</extends>
   <header>Form/Controls/Filter.hpp</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>EggSettings</class>
   <extends>QWidget</extends>
   <header>Form/Controls/EggSettings.hpp</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>comboBoxProfiles</tabstop>
  <tabstop>pushButtonProfileManager</tabstop>
  <tabstop>tabEggSelection</tabstop>
  <tabstop>textBoxEmeraldInitialAdvancesHeld</tabstop>
  <tabstop>textBoxEmeraldMaxAdvancesHeld</tabstop>
  <tabstop>textBoxEmeraldInitialAdvancesPickup</tabstop>
  <tabstop>textBoxEmeraldMaxAdvancesPickup</tabstop>
  <tabstop>textBoxEmeraldDelayHeld</tabstop>
  <tabstop>textBoxEmeraldDelayPickup</tabstop>
  <tabstop>textBoxEmeraldCalibration</tabstop>
  <tabstop>textBoxEmeraldMinRedraws</tabstop>
  <tabstop>textBoxEmeraldMaxRedraws</tabstop>
  <tabstop>comboBoxEmeraldMethod</tabstop>
  <tabstop>comboBoxEmeraldCompatibility</tabstop>
  <tabstop>pushButtonEmeraldGenerate</tabstop>
  <tabstop>tableViewEmerald</tabstop>
  <tabstop>textBoxRSFRLGSeedHeld</tabstop>
  <tabstop>textBoxRSFRLGSeedPickup</tabstop>
  <tabstop>textBoxRSFRLGInitialAdvancesHeld</tabstop>
  <tabstop>textBoxRSFRLGMaxAdvancesHeld</tabstop>
  <tabstop>textBoxRSFRLGInitialAdvancesPickup</tabstop>
  <tabstop>textBoxRSFRLGMaxAdvancesPickup</tabstop>
  <tabstop>textBoxRSFRLGDelayHeld</tabstop>
  <tabstop>textBoxRSFRLGDelayPickup</tabstop>
  <tabstop>comboBoxRSFRLGMethod</tabstop>
  <tabstop>comboBoxRSFRLGCompatibility</tabstop>
  <tabstop>pushButtonRSFRLGGenerate</tabstop>
  <tabstop>tableViewRSFRLG</tabstop>
 </tabstops>
 <resources/>
 <connections/>
 <slots>
  <slot>emeraldAnyAbility()</slot>
 </slots>
</ui>
